src/warnings/message.js

```diff
@@ -5,6 +5,6 @@ export default {
     browserManipulationsOnRemoteBrowser:     'The screenshot and window resize functionalities are not supported in a remote browser. They can function only if the browser is running on the same machine and in the same environment as the TestCafe server.',
     screenshotNotSupportedByBrowserProvider: 'The screenshot functionality is not supported by the "{providerName}" browser provider.',
     resizeNotSupportedByBrowserProvider:     'The window resize functionality is not supported by the "{providerName}" browser provider.',
-    maximizeNotSupportedByBrowserProvider:   'The maximize functionality is not supported by the "{providerName}" browser provider.',
+    maximizeNotSupportedByBrowserProvider:   'The window maximization functionality is not supported by the "{providerName}" browser provider.',
```

Comment:
oh, yeah, I forgot, should have checked... though @AndreyBelym said it's surely only a single file that needs to be changed.. but let's run them anyway